docs/concept-exercises.md

```diff
@@ -58,14 +58,14 @@ An exercise has the following files. In the browser they will show at the releva
 **Purpose:** Introduce the concept(s) that the exercise teaches to the student.
 
 - The information provided should give the student just enough context to figure out the solution themselves.
-- Links should be used sparingly if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.
-- The file should not give the student any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
-- Use the proper technical terms so that the student can easily search for more information.
-- If the exercise introduces new syntax, an example of the syntax should always be included; students should not need to search the web for examples of syntax.
+- Don't give any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
```

Comment:
I'd like this to be rules about the files, not rules about what people should do. I also think it would be good to have "dos" rather than "do nots". I think the language is less confrontational and more welcome. I'm happy to be disagreed with though.

So, as an example, this would become something more like:

```suggestion
- Only information that is needed to understand the fundamentals of the concept and solve the exercise should be provided. Extra information should be left for the `after.md`.
```

Reply:
I like it :)